src/main/java/org/cryptomator/common/settings/Settings.java

```diff
@@ -44,8 +46,8 @@ public class Settings {
 	static final String DEFAULT_KEYCHAIN_PROVIDER = SystemUtils.IS_OS_WINDOWS ? "org.cryptomator.windows.keychain.WindowsProtectedKeychainAccess" : SystemUtils.IS_OS_MAC ? "org.cryptomator.macos.keychain.MacSystemKeychainAccess" : "org.cryptomator.linux.keychain.SecretServiceKeychainAccess";
 	static final String DEFAULT_USER_INTERFACE_ORIENTATION = NodeOrientation.LEFT_TO_RIGHT.name();
 	static final boolean DEFAULT_SHOW_MINIMIZE_BUTTON = false;
-	static final String DEFAULT_LAST_UPDATE_CHECK = "2000-01-01";
-
+	static final Instant DEFAULT_LAST_UPDATE_REMINDER = Instant.ofEpochSecond(946681200); //2000-01-01T00:00:00Z
+	public static final Instant DEFAULT_LAST_SUCCESSFUL_UPDATE_CHECK = Instant.ofEpochSecond(946681200); //2000-01-01T00:00:00Z
```

Comment:
No need for seconds + comment, the API has you covered:

```suggestion
	static final Instant DEFAULT_LAST_UPDATE_REMINDER = Instant.parse("2000-01-01T00:00:00Z");
	public static final Instant DEFAULT_LAST_SUCCESSFUL_UPDATE_CHECK = Instant.parse("2000-01-01T00:00:00Z");
```